contracts/kip13/InterfaceIdentifier.go

```diff
@@ -4,34 +4,37 @@
 package kip13
 
 import (
+	"math/big"
 	"strings"
 
+	"github.com/klaytn/klaytn"
 	"github.com/klaytn/klaytn/accounts/abi"
 	"github.com/klaytn/klaytn/accounts/abi/bind"
 	"github.com/klaytn/klaytn/blockchain/types"
 	"github.com/klaytn/klaytn/common"
+	"github.com/klaytn/klaytn/event"
+)
+
+// Reference imports to suppress errors if they are not otherwise used.
+var (
+	_ = big.NewInt
+	_ = strings.NewReader
+	_ = klaytn.NotFound
+	_ = bind.Bind
+	_ = common.Big1
+	_ = types.BloomLookup
+	_ = event.NewSubscription
 )
 
 // InterfaceIdentifierABI is the input ABI used to generate the binding from.
 const InterfaceIdentifierABI = "[{\"constant\":true,\"inputs\":[{\"name\":\"interfaceID\",\"type\":\"bytes4\"}],\"name\":\"supportsInterface\",\"outputs\":[{\"name\":\"\",\"type\":\"bool\"}],\"payable\":false,\"stateMutability\":\"view\",\"type\":\"function\"}]"
 
 // InterfaceIdentifierBinRuntime is the compiled bytecode used for adding genesis block without deploying code.
-const InterfaceIdentifierBinRuntime = `0x`
+const InterfaceIdentifierBinRuntime = ``
 
-// InterfaceIdentifierBin is the compiled bytecode used for deploying new contracts.
-const InterfaceIdentifierBin = `0x`
-
-// DeployInterfaceIdentifier deploys a new Klaytn contract, binding an instance of InterfaceIdentifier to it.
-func DeployInterfaceIdentifier(auth *bind.TransactOpts, backend bind.ContractBackend) (common.Address, *types.Transaction, *InterfaceIdentifier, error) {
-	parsed, err := abi.JSON(strings.NewReader(InterfaceIdentifierABI))
-	if err != nil {
-		return common.Address{}, nil, nil, err
-	}
-	address, tx, contract, err := bind.DeployContract(auth, parsed, common.FromHex(InterfaceIdentifierBin), backend)
-	if err != nil {
-		return common.Address{}, nil, nil, err
-	}
-	return address, tx, &InterfaceIdentifier{InterfaceIdentifierCaller: InterfaceIdentifierCaller{contract: contract}, InterfaceIdentifierTransactor: InterfaceIdentifierTransactor{contract: contract}, InterfaceIdentifierFilterer: InterfaceIdentifierFilterer{contract: contract}}, nil
```

Comment:
Can you explain why did you removed this function?

Reply:
This file is generated by `abigen`. There's no contract implementation in `contracts/kip13`. The previous commit seemed to run `abigen` with another contract file that has identifier contract implementation. Currently, there's only an interface in the `contracts/kip13` dir.